dotCMS/src/main/java/com/dotmarketing/portlets/workflows/business/WorkflowAPIImpl.java

```diff
@@ -49,6 +49,7 @@
 import com.liferay.portal.language.LanguageUtil;
 import com.liferay.portal.model.User;
 
+import com.liferay.util.StringUtil;
 import org.osgi.framework.BundleContext;
 

```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'com.liferay.util.StringUtil'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)